README.md

```diff
@@ -183,13 +183,15 @@ network_config:
   enabled: true
 ```
 
-Once modification is complete, follow the steps below:
+**Note**: This configuration works with Agent 6.24.1+ and 7.24.1+. For older Agent versions, refer to [the public documentation][8] on how to enable system-probe.
+
+On Linux, once this modification is complete, follow the steps below if you installed an Agent version older than 6.18.0 or 7.18.0:
```

Comment:
Added this because on later versions, this should not be needed since the `datadog-agent-sysprobe` service is tied to the `datadog-agent` service.

Another option would be to automate these steps, but I'm not sure it's worth spending time on it given that these are old versions.